Doc/c-api/arg.rst

```diff
@@ -331,13 +331,13 @@ Other objects
 ``O&`` (object) [*converter*, *anything*]
    Convert a Python object to a C variable through a *converter* function.  This
    takes two arguments: the first is a function, the second is the address of a C
-   variable (of arbitrary type), converted to :c:type:`void \*`.  The *converter*
+   variable (of arbitrary type), converted to :c:expr:`void*`.  The *converter*
    function in turn is called as follows::
 
       status = converter(object, address);
 
    where *object* is the Python object to be converted and *address* is the
-   :c:type:`void\*` argument that was passed to the :c:func:`PyArg_Parse\*` function.
+   :c:expr:`void*` argument that was passed to the ``PyArg_Parse*`` functions.
```

Comment:
`:c:func:` not just changes the style and adds a link, but it also adds `()` after the function name. Please add explicit `()` in all sites where you changed `:c:func:` to `:c:expr:` or ``` `` `` ```.
```suggestion
   :c:expr:`void*` argument that was passed to the ``PyArg_Parse*()`` functions.
```